### ISSUE TITLE
Angular 13

### ISSUE THREAD

**Issue by rmpt** [rnadler/ng9-password-strength-bar#8]
Not working on angular 13 due to dependencies resolve.

**Comment by elsasslegend** [comment]
+1

**Comment by rnadler** [comment]
Use ```npm install --force```. This will turn the dependency errors into warnings. This worked for a new Angular 14 project.
Let me know if this resolves the issue. Thanks!

**Comment by rmpt** [comment]
yes, of course it will work like this, or with `--legacy-peer-deps` but the idea is to use it with no warnings or legacy dependencies.
